src/App.tsx

```diff
@@ -2,72 +1,29 @@
+import React, { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
```

Comment:
Verify that the `Person` type includes the `name`, `born`, and `died` properties, as these are used in the title display.